hw/ip/hmac/dv/cov/hmac_cov_excl.el

```diff
@@ -18,273 +18,169 @@ CHECKSUM: "3109872245 3727953184"
 INSTANCE: tb.dut.u_reg.u_msg_length_upper
 ANNOTATION: "[UNR]"
 Block 1 "0" "assign wr_en = de;"
-CHECKSUM: "2330220405 3466268034"
+CHECKSUM: "4294926009 3466268034"
 INSTANCE: tb.dut.u_tlul_adapter.u_rspfifo
-ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
-Block 17 "3894529251" "gen_normal_fifo.fifo_wptr <= (gen_normal_fifo.fifo_wptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
-Block 16 "1403547534" "gen_normal_fifo.fifo_wptr <= {(~gen_normal_fifo.fifo_wptr[(gen_normal_fifo.PTR_WIDTH - 1)]), {(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}};"
 Block 15 "1874817428" "if ((gen_normal_fifo.fifo_wptr[0] == (Depth - 1)))"
-Block 26 "3968208487" "gen_normal_fifo.fifo_rptr <= (gen_normal_fifo.fifo_rptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
-Block 25 "3085096792" "gen_normal_fifo.fifo_rptr <= {(~gen_normal_fifo.fifo_rptr[(gen_normal_fifo.PTR_WIDTH - 1)]), {(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}};"
-Block 24 "2927261095" "if ((gen_normal_fifo.fifo_rptr[0] == (Depth - 1)))"
 Block 32 "2823775765" "gen_normal_fifo.storage[0] <= wdata;"
 Block 30 "0" "assign gen_normal_fifo.storage_rdata = gen_normal_fifo.storage[0];"
-CHECKSUM: "2330220405 3466268034"
-INSTANCE: tb.dut.u_reg.u_socket.fifo_h.reqfifo
-ANNOTATION: "[UNR] ptr_width is 2, so it always reaches the previous condition"
-Block 17 "3894529251" "gen_normal_fifo.fifo_wptr <= (gen_normal_fifo.fifo_wptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
-Block 26 "3968208487" "gen_normal_fifo.fifo_rptr <= (gen_normal_fifo.fifo_rptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
-CHECKSUM: "2330220405 3466268034"
-INSTANCE: tb.dut.u_reg.u_socket.fifo_h.rspfifo
-ANNOTATION: "[UNR] ptr_width is 2, so it always reaches the previous condition"
-Block 17 "3894529251" "gen_normal_fifo.fifo_wptr <= (gen_normal_fifo.fifo_wptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
-Block 26 "3968208487" "gen_normal_fifo.fifo_rptr <= (gen_normal_fifo.fifo_rptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
-CHECKSUM: "2330220405 3466268034"
-INSTANCE: tb.dut.u_reg.u_socket.gen_dfifo[0].fifo_d.reqfifo
-ANNOTATION: "[UNR] ptr_width is 2, so it always reaches the previous condition"
-Block 17 "3894529251" "gen_normal_fifo.fifo_wptr <= (gen_normal_fifo.fifo_wptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
```

Comment:
These coverages are gone because in PR #884, fifo depth is 0, so previously FSM, branch coverages are no longer there (they are only for depth > 0).